docs/podlet/local_development.md

```diff
@@ -53,119 +53,92 @@ The first problem with the basic setup described above is that every time you ma
 
-_Use nodemon to run your server_
-
 ```bash
 npx nodemon server.js
 ```
 
 See the [nodemon docs](https://github.com/remy/nodemon#nodemon) for more information.
 
+Node also has a built-in `--watch` mode you can use:
+
+```bash
+node --watch server.js
+```
+
+See the [Node.js docs](https://nodejs.org/docs/v20.13.1/api/cli.html#--watch) for more information.
+
 ### Missing context headers
 
-When a podlet is being run in the context of a layout server, the layout server will send a number of Podium context headers with each request. If your podlet depends on these headers to work correctly then developing the podlet locally may be difficult or impossible out of the box.
+When a podlet is being run in the context of a layout server, the layout server will send a number of [Podium context](/docs/introduction/context) headers with each request. If your podlet depends on these headers to work correctly you need to turn on `development` mode.
 
 Consider a podlet with the following content route:
 
 ```js
 app.get(podlet.content(), (req, res) => {
-    const { mountOrigin } = res.locals.podium.context;
-    res.send(`<div>${mountOrigin}</div>`);
+  const { mountOrigin } = res.locals.podium.context;
+  res.send(`<div>${mountOrigin}</div>`);
 });
 ```
 
 This podlet will behave correctly when sent requests by a layout but it will throw an error if you try to visit `/` directly in your browser.
 
-To solve this, we provide a `development` mode. When enabled, you can set defaults for Podium context values that will be overwritten, and therefore not used, when requests are sent from the layout to the podlet.
+When `development` is enabled, you can set defaults for Podium context values that will be overwritten, and therefore not used, when requests are sent from the layout to the podlet.
 
 To enable this feature, pass `development: true` in the podlet constructor like so:
 
 ```js
 const podlet = new Podlet({
-    ...
-    development: true,
+  development: true,
 });
 ```
 
-Then rewriting our previous example we can provide sensible development defaults for context values:
+Podium includes some sensible defaults, but you can override them if you like.
 
 ```js
 podlet.defaults({
-    mountOrigin: 'http://localhost:7100',
-});
-
-app.get(podlet.content(), (req, res) => {
-    const { mountOrigin } = res.locals.podium.context;
-    res.send(`<div>${mountOrigin}</div>`);
-});
-```
-
-Even better, simply enabling `development` mode will result in some sane context defaults being provided, regardless of whether you call `.defaults` or not.
-
-So the following should work when `development` mode is on as well:
-
-```js
-app.get(podlet.content(), (req, res) => {
-    const { mountOrigin } = res.locals.podium.context;
-    res.send(`<div>${mountOrigin}</div>`);
+  locale: "nb-NO",
```

Comment:
would it make more sense for this to be english, since all the docs are in english etc? 🤔

Reply:
That's already the default though 😅

Reply:
aha! right, then perhaps it does make sense (or remove it 🤷 )